client/extensions/woocommerce/state/sites/error-reducer.js

```diff
@@ -9,7 +9,7 @@ import debugFactory from 'debug';
 import {
 	WOOCOMMERCE_API_SET_ERROR,
 	WOOCOMMERCE_API_CLEAR_ERROR,
-} from '../action-types';
+} from 'woocommerce/state/action-types';
```

Comment:
I'd like to point out that changes like this in reducers won't work until #14731 is merged.